.github/dependabot.yaml

```diff
@@ -1,9 +1,29 @@
 version: 2
 
 updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    groups:
+      flux-deps:
+        patterns:
+          - "github.com/fluxcd/*"
+    allow:
+      - dependency-name: "github.com/fluxcd/*"
```

Comment:
> Use the allow option to customize which dependencies are updated. This applies to both version and security updates.

From https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#allow.

Not sure if that's what we want.

Reply:
I want only fluxcd updates, I don't see how I can do that with ignore unless we list all deps in there.